src/components/App.js

```diff
@@ -1,28 +1,39 @@
-import { Layout } from 'antd';
+import { Layout, notification } from 'antd';
 import PropTypes from 'prop-types';
 import React from 'react';
 import { BrowserRouter, Link } from 'react-router-dom';
 import { connect } from 'react-redux';
 
-import Error from './Error';
-import { getError } from 'console/state/auth/selectors';
-
 import AuthButton from 'console/components/auth/AuthButton';
 import NavBar from 'console/components/navigation/NavBar';
 import QueryActions from 'console/components/data/QueryActions';
 import QueryAuth0 from 'console/components/data/QueryAuth0';
 import AppRouter from 'console/components/AppRouter';
 import CircleLogo from 'console/components/svg/CircleLogo';
+import { getError } from 'console/state/auth/selectors';
+
 const { Header } = Layout;
 
 @connect((state, props) => ({
-  error: getError(state),
+  authError: getError(state),
```

Comment:
Much better! I'm sorry for not pointing this out. I thought this `error` was going to be used for a bunch of other types of errors. Now it all of a sudden makes sense to hardcode "Authentication Error".